### PR TITLE
Shorten dvara start up time

### PR DESCRIPTION
**Pull request by bbaja42**
If we start with 10 replica sets, delay can be 10 \* 5 seconds, 50 seconds. With this, the delay is reduced to max 5 seconds.

### CODE REVIEW

File: rs_state.go [hunk 1]
```diff
@@ -22,20 +22,21 @@ type ReplicaSetState struct {
 
 // NewReplicaSetState creates a new ReplicaSetState using the given address.
 func NewReplicaSetState(username, password, addr string) (*ReplicaSetState, error) {
+	TIMOUT := 500 * time.Millisecond
```
Comment (josler) [inline rs_state.go]: add this as a const at the top?

``` go
const(
    errNotReplSet = "not running with --replSet"
    timeout = 500 * time.Millisecond
)
```

`TIMOUT` -> `timeout` too :)

**Comment by dannyfallon** [comment]
Is that TIMO....oh, you got there already :smile:

**Comment by bbaja42** [comment]
Alex spotted it :)

**Comment by agerlic** [comment]
:sheep:

File: rs_state.go [hunk 2]
```diff
@@ -22,20 +22,21 @@ type ReplicaSetState struct {
 
 // NewReplicaSetState creates a new ReplicaSetState using the given address.
 func NewReplicaSetState(username, password, addr string) (*ReplicaSetState, error) {
+	TIMEOUT := 500 * time.Millisecond
```
Comment (josler) [inline rs_state.go]: const!

Comment (bbaja42) [inline rs_state.go]: Uuh, nice. I didn't know golang had this keyword. Thnx.

Comment (mariokostelac) [inline rs_state.go]: it does, but don't event think about using it as in c++.

**Pull request merged**